components/omnibox/browser/brave_bookmark_provider.cc

```diff
@@ -19,4 +21,46 @@ void BraveBookmarkProvider::Start(const AutocompleteInput& input,
     return;
   }
   BookmarkProvider::Start(input, minimal_changes);
+
+  if (input.text().empty() || matches_.empty()) {
+    return;
+  }
+
+  // We need to bump the relevance of the bookmark if we ever want it to rank
+  // high enough to be the default match.
+  constexpr int kContainsQueryBump = 350;
+  bool modified = false;
+
+  auto lower_text = base::ToLowerASCII(input.text());
+  for (auto& match : matches_) {
+    if (match.from_keyword) {
+      continue;
+    }
```

Comment:
q) it can be true from bookmarks matches?

Reply:
I figured if there's a keyword match (i.e. we have a `@bookmarks`) its probably fine to do the default thing